examples/minimal/app.js

```diff
@@ -3,13 +3,25 @@
  * Demonstrates the new ECS architecture with plugins
  */
 
-import { Engine } from '../../src/core/engine.js';
+/*import { Engine } from '../../src/core/engine.js';
```

Comment:
i need to remove this excluded code.

Reply:
done with https://github.com/AR-js-org/AR.js-core/pull/8/commits/552155635b42dcc1a68b82759b0e1a112ec168e2